### DISCUSSION TITLE
Status with Ubuntu Line IN red

### DISCUSSION THREAD

**Discussion by AlJeske** [NightscoutFoundation/xDrip#3354] [General]
STATUS: Partial copy:

>  │ Ubuntu: 20.04.5 LTS  

 
 I'm trying to update my nightscout in Google cloud and everything was fine except the Ubuntu: line was in red. And I checked the NS version and it was still the old version.  I presently get Xdrip+ to send BG info to NS but I would like more info to be sent besides Glucose. As I understand the new version of Aaps that I am using can only respond to NS v15. Hence the upgrade that I tried to do. ( tried a screen shot but I couldn't transfer it to this discussion so I copied the part highlighted in red.

**Comment by Navid200** [comment]
You can verify that you have upgraded Nightscout by bringing it up in a browser and looking under the menu and scrolling all the way down.

The red item on the status page is ok for now.
It will be fixed soon.  But, it has no negative impact.

**Comment by AlJeske** [comment]
Still shows this:
nightscout
version 14.2.6
--Al Jeske
________________________________

**Reply by Navid200** [reply]
Have you restarted the server after the update?
You need to.

**Comment by AlJeske** [comment]
No, not yet. What is the server and how is it restarted?

**Reply by Navid200** [reply]
https://navid200.github.io/xDrip/docs/Nightscout/Restart.html

**Comment by AlJeske** [comment]
Thanks, I'll check it out!

**Comment by AlJeske** [comment]
It says "Connection Failed
You cannot connect to the VM instance because of an expected error. Wait and try again....." 
I tried again with same results.  Tried to open up Nightscout  and it said "502 Bad Gateway"

**Reply by Navid200** [reply]
Run phase 1.

**Reply by Navid200** [reply]
It's on the Google Cloud Setup submenu.
https://navid200.github.io/xDrip/docs/Nightscout/Menu.html

**Comment by AlJeske** [comment]
Thanks, I checked it out! It still doesn't work. Sending a SS of results of
opening NS.

**Reply by Navid200** [reply]
I don't know what "sending a SS of ... " means.

There is a lot of information here: https://navid200.github.io/xDrip/docs/Nightscout/GoogleCloud.html

And this goes over troubleshooting steps:
https://navid200.github.io/xDrip/docs/Nightscout/Troubleshooting.html

Go over that and see if it helps.
If it doesn't, I need to know what you did following all those instructions and what responses you got.
In that case, the first thing I will need is a snapshot of the status page.

**Reply by Navid200** [reply]
As an alternative, there is the option of paying someone to do the work for you.  The fee could be lower than $5 a month.
Nightscout pro is only one option.  There are many more.

If you don't want to pay, you will have to do the work yourself.
I can only provide support.

**Comment by AlJeske** [comment]
I want to keep trying. SS is screen shot. I thought I sent it.🥴 I'll try
sending it again.

**Reply by Navid200** [reply]
You shouldn't respond to an email in order to post here.
Please log into GitHub and then you can attach an image when creating or editing your post.

**Reply by AlJeske** [reply]
Trouble shooting Google cloud NS 
	Machine has not stopped running 
	Update platform
	502 Bad Gateway
	Reboot server
	Connection Failed
	Waited, Opened NS **v15.0.2** (might not have followed the instructions exactly—the wait part???)
--Very Happy!!! Now I'm going to try using Aaps to upload to NS.  Do I need to stop sending to NS from xDrip+ before I try to connect Aaps?

**Reply by Navid200** [reply]
I'm happy things have moved forward.

I'm afraid I am not the right person to advise about AAPS.

@olorinmaia Where is the best source of info for @AlJeske in order to get AAPS set up with his Nightscout?

**Reply by olorinmaia** [reply]
https://androidaps.readthedocs.io/en/latest/Installing-AndroidAPS/Nightscout.html#nightscout

This and #WeAreNotWaiting on discord https://discord.com/invite/KwQ45fRj

**Reply by olorinmaia** [reply]
And this https://androidaps.readthedocs.io/en/latest/Configuration/xdrip.html#xdrip-settings

**Comment by Navid200** [comment]
The Ubuntu update has now been resolved.
The next time you update your platform (https://navid200.github.io/xDrip/docs/Nightscout/NS_SyncExecutables.html), the 20.04.5 line on your status page will change to 20.04.6.

This is not a critical update and you don't have to rush to do it.
The Ubuntu update you are missing is not an update that compromises anything you are doing with your virtual machine.
The next time you update, everything will be the same as all the other sites.
